ddtrace/internal/pack_template.h

```diff
@@ -645,6 +645,9 @@ static inline int msgpack_pack_raw(msgpack_packer* x, size_t l)
     if (l < 32) {
         unsigned char d = 0xa0 | (uint8_t)l;
         msgpack_pack_append_buffer(x, &TAKE8_8(d), 1);
+    } else if (l < 256) {
+        unsigned char buf[2] = {0xd9, (uint8_t)l};
+        msgpack_pack_append_buffer(x, buf, 2);
```

Comment:
is this missing from our current implementation? what are the side effects of this change on the existing encoder?

Reply:
Hmm it is missing from the current and will be used when encoding strings but there shouldn't be any effect (other than a different msgpack prefix being used for strings that are between 32 and 256 bits. In theory I think this should yield some slight performance benefit 
<img width="679" alt="image" src="https://user-images.githubusercontent.com/6321485/144497085-498aa8ea-858b-49cd-86e9-cf31ab9348bb.png">

(saves one byte per string that is < 256 bits in length)